scripts/prepare_sector_network.py

```diff
@@ -208,19 +208,33 @@ def add_hydrogen(n, costs):
             x=n.buses.loc[list(nodes)].x.values,
             y=n.buses.loc[list(nodes)].y.values,
         )

```

Comment:
Please also add H2O buses. This could be done very similar to: https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/scripts/prepare_sector_network.py#L196